ios/browser/api/wallet/brave_wallet_service_factory.cc

```diff
@@ -0,0 +1,47 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
the factory should go in the same location as desktop so `ios/browser/brave_wallet/brave_wallet_service_factory`